src/goto-programs/restrict_function_pointers.cpp

```diff
@@ -579,6 +579,8 @@ void function_pointer_restrictionst::write_to_file(
   }
 
   function_pointer_restrictions_json.output(outFile);
+  // output does not have a newline at the end
```

Comment:
⛏️ I'd say a better comment here would be "Ensure the outFile ends with a newline character"

Reply:
Good stuff. I've implemented this.